dallinger/recruiters.py

```diff
@@ -727,7 +727,8 @@ def notify_duration_exceeded(self, participants, reference_time):
                 self._send_notification_missing_rest_notification_for(participant)
                 unsubmitted.append(summary)
 
-        if unsubmitted:
+        disable_hit = self.config.get("disable_when_duration_exceeded", True)
```

Comment:
This isn't the only place in Dallinger like this, but how come the default value is hard-coded here instead of referring to `global_config_defaults.txt`? Or am I misunderstanding how this works?

Reply:
I'm +1 on @pmcharrison's concern. We had an initiative a couple years ago to move defaults to the default config files so they wouldn't be scattered around, and it seems like we should stick with that goal (though there is certainly also an argument to be made for including the default at the one and only place it's read).

Reply:
This is definitely redundant with the global setting and I can remove it if desired. I tend to think of it as a backup/failsafe for the config default (preventing failure if somehow the default were unset), but it's most likely unnecessary.

Reply:
Fixed.